flavors/template-Exasol-all-python-3.12-conda/flavor_base/udfclient_deps/packages/conda_packages

```diff
@@ -0,0 +1,4 @@
+libprotobuf|6.31.1
```

Comment:
likely due to latest vs compatibility:
- libprotobuf 6.32.1
- nss 3.116

Reply:
Can't update protobuf (same as above).
But new version for nss work